src/patternfly/components/Table/table.scss

```diff
@@ -1089,10 +1092,6 @@
   .#{$table}__thead & {
     border-block-end: 0;
   }
-
-  &.pf-m-border-row {
-    border-block-end: var(--#{$table}--border-width--base) solid var(--#{$table}--BorderColor);
-  }
```

Comment:
Just want to make sure this is intentional? If so can you give a summary of why? Looks like we'll need a react update for this, too.